test/functional/wallet_balance.py

```diff
@@ -285,5 +287,30 @@ def test_balances(*, fee_node_1=0):
         assert_equal(tx_info['lastprocessedblock']['height'], prev_height)
         assert_equal(tx_info['lastprocessedblock']['hash'], prev_hash)
 
+        self.log.info("Test that the balance is updated by an import that makes an untracked output in an existing tx \"mine\"")
+        default = self.nodes[0].get_wallet_rpc(self.default_wallet_name)
+        self.nodes[0].createwallet("importupdate")
```

Comment:
Is this wallet watch-only (per commit description) ?

Reply:
It is not. That part of the test no longer exists after #28710. I've updated the commit message.